### PR TITLE
WIP: form/prop/tag tombstones

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

File: synapse/lib/layer.py [hunk 1]
```diff
@@ -1161,6 +1161,14 @@ async def _initLayerStorage(self):
...
         self.onfini(self.dataslab)
 
+        # various properties/tags/data about a node
+        # 00 - formvalu (buid, 00) = (form, valu, stortype)
+        # 01 - property (buid, 01, penc) = (valu, stortype)
+        # 02 - tag (buid, 01, tenc) = valu
+        # 09 - form
+        # 0a - form tombstone (buid, 0a) = <editinfo>
+        # 0b - prop tombstone (buid, 0b, penc) = <editinfo>
+        # 0c - tag tombstone (buid, 0c, tenc) = <editinfo>
```
Comment (vEpiphyte) [inline synapse/lib/layer.py:1171]: Other things to consider off hand include the following:

Tag Props
Light Edges
Node Data

Comment (invisig0th) [reply synapse/lib/layer.py:1171]: yep thanks, was def planning to include them eventually :)

File: synapse/lib/layer.py [hunk 2]
```diff
@@ -2153,73 +2211,52 @@ async def iterLayerNodeEdits(self):
...
             if flag == 9:
-                if not nodeedits[0] == buid:
-                    form = lval.decode()
-                    nodeedits = (buid, form, [])
-
-                elif nodeedits[1] is None:
-                    form = lval.decode()
-                    nodeedits = (nodeedits[0], form, nodeedits[2])
-
+                nodeedits[1] = lval.decode()
```
Comment (vEpiphyte) [inline synapse/lib/layer.py:2244]: ???

Comment (invisig0th) [reply synapse/lib/layer.py]: we're more efficiently constructing the `nodeedits` and this is the line that updates the form. see the removed lines above.

**Pull request closed without merging**